checker/src/main/java/org/checkerframework/checker/signedness/SignednessChecker.java

```diff
@@ -11,6 +12,18 @@
  *
  * @checker_framework.manual #signedness-checker Signedness Checker
  */
+// Character and char are omitted here because they are always @Unsigned, and the user is not
```

Comment:
So this makes it so there will be (almost) no warnings about `char`s.  I don't understand why this is safe.

Reply:
Since the programmer cannot annotate `char`s, there isn't much that can go wrong.
An assignment like
```java
  int i = myChar;
```
is fine regardless of the signedness of `i`.
A narrowing like
```java
  byte b = (byte) myChar;
```
is also fine regardless of the signedness of `b`.

There used to be tests that used signed characters, but that is not sensible and I removed those tests.  The checker does issue an error if the programmer writes `@Signed char` as a type.